yb-voyager/src/migassessment/assessmentDB.go

```diff
@@ -146,6 +147,15 @@ func InitAssessmentDB() error {
 			redundant_ddl TEXT,
 			existing_ddl TEXT,
 			PRIMARY KEY(redundant_schema_name,redundant_table_name,redundant_index_name));`, REDUNDANT_INDEXES),
+		fmt.Sprintf(`CREATE TABLE IF NOT EXISTS %s (
+			schema_name TEXT,
+			table_name TEXT,
+			column_name TEXT,
+			null_frequency REAL,
```

Comment:
this should be `null_frac`, as it's not really "frequency". Pls also change it in the query.